code/modules/paperwork/photography.dm

```diff
@@ -336,6 +341,28 @@
 
 
 
+/obj/item/device/camera/proc/camera_get_assassination_results(turf/the_turf)
```

Comment:
I wonder if this is going to be a small lagbomb wither higher resolution cameras. Probably not (those `for` loops are at best one iteration every time), but if it is, I would just take the list of targets of the agent, assess their status (dead or alive) and check if their coordinates match the location the photo was taken.

Reply:
> but if it is, I would just take the list of targets of the agent, assess their status (dead or alive) and check if their coordinates match the location the photo was taken.

That's even more convoluted and inaccurate. This check is not only skipped when there are no assassination objectives around, but the amount of checks done is negligible compared to everything else done when making a photo, starting with all the icon operations.